rollup.config.js

```diff
@@ -55,8 +56,37 @@ export default {
        * we're externalizing all "node_modules", it doesn't exist.
        * This little hack makes sure we're using "node_modules" instead of what the plugin expects.
        */
-      inject(cssVariableName) {
-        return `import styleInject from 'style-inject';\nstyleInject(${cssVariableName}, { insertAt: 'top' });`;
+      inject: function (cssVariableName) {
+        const hashValue = `s_id-${version}-${performance.now()}`.replaceAll(".", "_");
+        return `function styleInject(css, { insertAt } = {}) {
+    const head = document.head || document.getElementsByTagName('head')[0]
+    const id = "${hashValue}";
+    const styleExists = head.querySelector("#${hashValue}");
+    if(styleExists) return;
+    
+    const style = document.createElement('style');
+    style.id = id;
+  
+    if (insertAt === 'top') {
+      if (head.firstChild) {
+        head.insertBefore(style, head.firstChild)
+      } else {
+        head.appendChild(style)
+      }
+    } else {
+      head.appendChild(style)
+    }
```

Comment:
Can be simplified to:
```suggestion
    if (head.firstChild) {
      head.insertBefore(style, head.firstChild);
    } else {
      head.appendChild(style);
    }
```

And since we now control the implementation of `styleInject`, we can simplify it by not passing `injectAt` at all.